src/app/core/services/testing/test-user.service.ts

```diff
@@ -18,16 +18,16 @@ export class TestUserService extends UserService {
   update(user: User): Observable<User> {
```

Comment:
should you Partial<userService> instead of create new file 
https://angular.io/guide/testing-components-scenarios#provide-service-test-doubles